torch/_dynamo/eval_frame.py

```diff
@@ -666,13 +620,13 @@ def guard_export_print(guards):
     # TODO(voz): Do we want a decorator for this?
     reset()
     return ExplainOutput(
+        graphs,
         graph_count,
         graph_break_count,
+        break_reasons,
         op_count,
-        out_guards,
-        graphs,
         ops_per_graph,
-        break_reasons,
+        out_guards,
         compile_time,
     )
```

Comment:
why the motion here

Reply:
My OCD kicked in and I updated the order of the ExplainOutput attributes to be grouped together more logically :)